### DISCUSSION TITLE
change background color to black

### DISCUSSION THREAD

**Discussion by gobimahato** [skbolton/embark#61] [General]
I want to change background color to pure dark from purple default bg color

**Comment by skbolton** [comment]
After you set the colorscheme to embark you can manually adjust highlight groups to whatever you want.

Background is controlled by `Normal` but you'll probably then need to adjust the gutters and popup windows too. Those are things like `LineNr`, `CursorLineNr`, `SignColumn`, `FoldColumn`, `Pmenu*`.
